### ISSUE TITLE
Subscription "Message must be a JSON string"

### ISSUE THREAD

**Issue by xxSkyy** [mercurius-js/mercurius#556]
Hey, I'm not doing pull request because I've got fork of modified version of mercurius with other customizations.
When today I fresh installed mercurius (from this repo) and I've had error from title on response. Checked it and it was because buffer looked like
` <Buffer 7b 22 74 79 70 65 22 3a 22 63 6f 6e 6e 65 63 74 69 6f 6e 5f 69 6e 69 74 22 2c 22 70 61 79 6c 6f 61 64 22 3a 7b 7d 7d>,
  false`
and this `,false` did problem while parsing. On installed like month ago server subs worked fine but on clean  node_modules and lockfile and fresh install error appeared. 
I don't know what causing it but here's my fix for it.
```js
/// lib/subscription-connection.js : 76
  async handleMessage(message) {
    let data
    try {
      let msg = message.toString()
      if(msg.slice(-6) === ",false") {
        msg = msg.substring(0, msg.length - 6)
      }
      data = sJSON.parse(msg)
    } catch (e) {
      this.sendMessage(GQL_ERROR, null, "Message must be a JSON string")
      return
    }
```

**Comment by mcollina** [comment]
Thanks for the message. I'm not sure how or why this problem happens, or why you felt the urge to fork. We need a bit more to land this change:

1. an example to reproduce the problem
2. a test

It seems you are not interested in contributing back, so I'm going to close this issue. If this problem appears for others too, we'll know it's here for fixing. Let us know if you change your mind and would like to PR this in.

**Comment by PabloSzx** [comment]
@mcollina this issue is of the v8 of ws that has a big amount of breaking changes https://github.com/websockets/ws/releases/tag/8.0.0 and it was released as a patch in v3.2.1 of fastify-websocket https://github.com/fastify/fastify-websocket/releases/tag/v3.2.1
